src/library-authoring/collections/LibraryCollectionPage.tsx

```diff
@@ -200,7 +204,7 @@ const LibraryCollectionPage = () => {
       extraFilter={[`context_key = "${libraryId}"`, `collections.key = "${collectionId}"`]}
       overrideQueries={{ collections: collectionQuery }}
```

Comment:
We can override collections query to an empty object as we don't need it here.

Reply:
Just a note: an empty object query returns all documents from the index. I replaced it with: `{ limit: 0}`